### DISCUSSION TITLE
Unable to `go get github.com/dexidp/dex/cmd/dex`

### DISCUSSION THREAD

**Discussion by pieterlexis** [dexidp/dex#2004] [Q&A] [answered]
Hi folks,

I'd like to have a local dev env with dex, but I can't seem to be able to 'just' `go get` the dex command.

I've tried both `go get -v github.com/dexidp/dex/cmd/dex` and `go get -v github.com/dexidp/dex/cmd/dex@86ea49173cab85eb408956d46aa5bcd3fe0a1262` (today's master commit). I get compile errors:

```
# go.etcd.io/etcd/clientv3/balancer/resolver/endpoint
../../go/pkg/mod/go.etcd.io/etcd@v0.0.0-20191023171146-3cf2f69b5738/clientv3/balancer/resolver/endpoint/endpoint.go:114:78: undefined: resolver.BuildOption
../../go/pkg/mod/go.etcd.io/etcd@v0.0.0-20191023171146-3cf2f69b5738/clientv3/balancer/resolver/endpoint/endpoint.go:182:31: undefined: resolver.ResolveNowOption
# go.etcd.io/etcd/clientv3/balancer/picker
../../go/pkg/mod/go.etcd.io/etcd@v0.0.0-20191023171146-3cf2f69b5738/clientv3/balancer/picker/err.go:37:44: undefined: balancer.PickOptions
../../go/pkg/mod/go.etcd.io/etcd@v0.0.0-20191023171146-3cf2f69b5738/clientv3/balancer/picker/roundrobin_balanced.go:55:54: undefined: balancer.PickOptions
```

Building from git (cloning and running `make build`) does work. But that doesn't really fit the flow of being able to get the tools I need in my project.

Is there a magic invocation to make this work, or is dex simply not installable via `go get` outside of the repo?

**Comment by nabokihms** [comment, marked as answer]
Hello! I think it is better to consider dex is not installable via go get.  [In this post](https://github.com/dexidp/dex/issues/1857#issuecomment-723507311) you can find the best explanation so far why it shouldn't be installed via go get.
